src/snapred/backend/dao/request/ReductionRequest.py

```diff
@@ -0,0 +1,16 @@
+from typing import List, Literal, Union
+
+from pydantic import BaseModel
+
+from snapred.backend.dao.state.FocusGroup import FocusGroup
+
+
+class ReductionRequest(BaseModel):
+    runNumber: str
+    useLitemode: bool
+    focusGroup: List[FocusGroup]
```

Comment:
The user doesnt actually choose the groups for reduction, it just reduces for each group applicable to a state.

Reply:
This is now 
``` python
focusGroup:  Union[Optional[FocusGroup], List[FocusGroup]
```
with the intention that this property will be set inside the reduction service after it pulls all the groupings.  The alternative is creating a bunch of single-use requests.